includes/razorpay-webhook.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
+
+require_once __DIR__.'/../razorpay-sdk/Razorpay.php';
+use Razorpay\Api\Api;
+
+class RZP_Webhook
+{
+    protected $razorpay;
+    protected $api;
+
+    function __construct()
+    {
+        $this->razorpay = new WC_Razorpay();
+
+        $this->api = $this->razorpay->getRazorpayApiInstance();
+    }
+
+    public function process()
+    {
+        $post = file_get_contents('php://input');
+
+        $data = json_decode($post, true);
```

Comment:
@captn3m0 If it is empty, it will return null. But yeah, if it isn't a valid json, it'll break. 

Either ways, `json_last_error` will be anything but 0 if json_decode failed for whatever reason. So I'll add that.